### PR TITLE
Migrate vstudio-extension to Prow

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Migrate VStudio extension to Prow  

**Related issue(s)**

See [#1](https://github.com/kyma-incubator/vstudio-extension/issues/1)

### CODE REVIEW

File: prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository-incubator: "true"
+
+presubmits: # runs on PRs
+  kyma-incubator/vstudio-extension:
+    - name: vstudio-extension
+      always_run: true
```
Comment (aszecowka) [inline prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml]: do we need to specify always_run always when run_if_changed is not specifed? If so, should we review other jobs if always_run is set?

File: prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      branches:
+        - master
+      <<: *job_template
+      labels:
+        <<: *job_labels_template
+        preset-build-pr: "true"
+
+postsubmits:
+  kyma-incubator/vstudio-extension:
+    - name: vstudio-extension
```
Comment (aszecowka) [inline prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml]: should we define always_run for postsubmits?

File: prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml (hunk 1 above)
Comment (suleymanakbas91) [reply prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml]: Yes, it seems so: https://github.com/kubernetes/test-infra/blob/fd04fb95d09df002e87df9f137b199ac4b6cac77/prow/jobs.md#how-to-configure-new-jobs

File: prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml (hunk 2 above)
Comment (suleymanakbas91) [reply prow/jobs/incubator/vstudio-extension/vstudio-extension.yaml]: No, it is a property of presubmit job type: https://github.com/kubernetes/test-infra/blob/050e7f5e2cd684ce12e9d382363c84f2dcdcd7ab/prow/config/jobs.go#L118

**Pull request merged**